### ISSUE TITLE
a11y: Table td scope

### ISSUE THREAD

**Issue by jschuler** [patternfly/patternfly-react#5199]
```
Expected the HTML found at $('td[scope=""]') to have no violations:
    <td scope="" class="" data-label="" data-key="6"></td>
    Received:
    "scope attribute should be used correctly (scope-attr-valid)"
    Fix all of the following:
      In HTML 5, scope attributes may only be used on table header elements (<th>)
      The value of the scope attribute may only be 'row' or 'col'
```
